packages/graphql/tests/performance/databaseQuery/TestRunner.ts

```diff
@@ -48,6 +48,7 @@ export class TestRunner {
             } catch (err) {
                 console.error("Error running test", test.filename, test.name);
                 console.warn(err);
+                results.push({ name: test.name, error: `${err}`, file: test.filename, type: "graphql" });
```

Comment:
Is the use of 

```
`${err}`
```

to convert this to a string? If so, it might be more clear to use `String(err)`